lib/VeresOneDriver.js

```diff
@@ -101,7 +111,7 @@ class VeresOneDriver {
    * @returns {Promise<object>} Resolves with the fetched or constructed DID
    *   Document.
    */
-  async get({did, url} = {}) {
+  async getFromLedger({did, url} = {}) {
```

Comment:
To help avoid anyone calling this and then us having to fix it:

```suggestion
  async _getFromLedger({did, url} = {}) {
```